File: CarDealerWebAPI/Core.CarDealer/Models/FuelType.cs
using System;
using System.Collections.Generic;

namespace Core.CarDealer.Models
{
    public partial class FuelType
    {
        public FuelType()
        {
            Cars = new HashSet<Car>();
        }

        public int Id { get; set; }
        public string? Name { get; set; }

        public virtual ICollection<Car> Cars { get; set; }
    }
}

File: CarDealerWebAPI/CarDealer.Client/ClientApp/src/app/Models/FuelType.ts




export class FuelType
{
    private _id!: string;
    public get id(): string {
        return this._id;
    }
    public set id(value: string) {
        this._id = value;
    }
    private _name!: string;
    public get name(): string {
        return this._name;
    }
    public set name(value: string) {
        this._name = value;
    }
}